includes/definitions/ciscoair.yaml

```diff
@@ -0,0 +1,50 @@
+os: ciscoair
+text: 'Cisco IOS'
```

Comment:
Do these devices run Cisco IOS?

Reply:
Yes, it does.

#show version
Cisco IOS Software, C1140 Software (C1140-K9W7-M), Version 15.3(3)JD11, RELEASE SOFTWARE (fc1)